spec/checkPullRequestJobSpec.js

```diff
@@ -0,0 +1,401 @@
+require('dotenv').config();
+const { createProbot } = require('probot');
+// The plugin refers to the actual app in index.js.
+const oppiaBot = require('../index');
+const checkPullRequestJobModule = require('../lib/checkPullRequestJob');
+const apiForSheetsModule = require('../lib/apiForSheets');
+const checkPullRequestLabelsModule = require('../lib/checkPullRequestLabels');
+const checkPullRequestBranchModule = require('../lib/checkPullRequestBranch');
+const checkWIPModule = require('../lib/checkWipDraftPR');
+const scheduler = require('../lib/scheduler');
+
+let payloadData = JSON.parse(
+  JSON.stringify(require('../fixtures/pullRequestPayload.json'))
+);
+const {SERVER_JOBS_ADMIN} = require('../userWhitelist.json');
+
+describe('Pull Request Job Spec', () => {
+  /**
+   * @type {import('probot').Probot} robot
+   */
+  let robot;
+
+  /**
+   * @type {import('probot').Octokit} github
+   */
+  let github;
+
+  /**
+   * @type {import('probot').Application} app
+   */
+  let app;
+
+  const newJobFileObj = {
+    sha: 'd144f32b9812373d5f1bc9f94d9af795f09023ff',
+    filename: 'core/domain/exp_jobs_one_off.py',
+    status: 'added',
+    additions: 1,
+    deletions: 0,
+    changes: 1,
+    blob_url:
+      'https://github.com/oppia/oppia/blob/67fb4a973b318882af3b5a894130e110d7e9833c/core/domain/exp_jobs_one_off.py',
+    raw_url:
+      'https://github.com/oppia/oppia/raw/67fb4a973b318882af3b5a894130e110d7e9833c/core/domain/exp_jobs_one_off.py',
+    contents_url:
+      'https://api.github.com/repos/oppia/oppia/contents/core/domain/exp_jobs_one_off.py?ref=67fb4a973b318882af3b5a894130e110d7e9833c',
+    patch: '@@ -0,0 +1 @@\n+# Testing job pushes',
+  };
+
+  const secondNewJobFileObj = {
+    sha: 'd144f32b9812373d5f1bc9f94d9af795f09023ff',
+    filename: 'core/domain/exp_jobs_oppiabot_off.py',
+    status: 'added',
+    additions: 1,
+    deletions: 0,
+    changes: 1,
+    blob_url:
+      'https://github.com/oppia/oppia/blob/67fb4a973b318882af3b5a894130e110d7e9833c/core/domain/exp_jobs_oppiabot_off.py',
+    raw_url:
+      'https://github.com/oppia/oppia/raw/67fb4a973b318882af3b5a894130e110d7e9833c/core/domain/exp_jobs_oppiabot_off.py',
+    contents_url:
```

Comment:
Can we use this to obtain the content of the file?

Reply:
Probably
The changes in the file are in the `patch`

Reply:
Ok, so we can use `patch` for finding new jobs being added in an existing file, right?